pkg/rtc/participant.go

```diff
@@ -42,6 +43,7 @@ type ParticipantParams struct {
 	ThrottleConfig  config.PLIThrottleConfig
 	EnabledCodecs   []*livekit.Codec
 	Hidden          bool
+	Logger          logr.Logger
```

Comment:
if we want to keep the shortcuts handy, this could also be a logger.WithLogger instance?  maybe we should just call it logger.Logger, as a type alias.